backend/app/schemas/session.py

```diff
@@ -14,3 +14,7 @@ class SessionOut(SchemaBase):
     id: str
     host_id: str
     quiz: QuizOut
+
+class UserSessionRankingOut(SchemaBase):
```

Comment:
Do we want to allow a lecturer to preview a user's solution? If so returning a solution along with the score makes sense.

Reply:
Probably, it is a good idea